railties/test/railties/engine_test.rb

```diff
@@ -879,6 +879,17 @@ def index
       assert Bukkits::Engine.config.bukkits_seeds_loaded
     end
 
+    test "jobs are ran inline while loading seeds" do
+      app_file "db/seeds.rb", <<-RUBY
+        ActiveJob::Base.queue_adapter.class.name
+      RUBY
+
+      boot_rails
+
+      assert_equal ActiveJob::QueueAdapters::InlineAdapter,  Rails.application.load_seed
```

Comment:
I think this doesn't do what you want. `Rails.application.load_seed` will return true.